Problem bei Informationssystem-Suche im klassischen Cleint	"Bei der ""Suche nach Informationssystemen"" im klassischen Client bekommen wir nach dem Update auf Version 5.5 eine Exception

""Exception while calling bridge#get class: de.iteratec.iteraplan.model.InformationSystemRelease path: runtimePeriod.runtimePeriod.start""



Version:

© iteratec GmbH 2017    Build ID: Corporate Edition v5.5.0-r26900Lizensiert für INTER Versicherungen



Vorgehen: Auswahl ""Informationssystem"" auf der Startseite, eingabe ""SAP"" im Eingabefeld, ""Suchen""



Über ""Auswertungen""->""Tabellarische Auswertungen"" lässt sich eine vergleichbare Suche problemlos absetzen."		Fault	iteraplan Support	17/May/17 2:26 PM	17/May/17 2:53 PM														"17/May/17 2:26 PM;Markus.Bon@inter.de;

!Meldung im Client.PNG|thumbnail!

[^catalina.out] _(217 kB)_

[^iteraplan-5.5.0.log] _(381 kB)_"	"17/May/17 2:41 PM;cst;Sehr geehrter Herr Dr. Bon,
sehr geehrter Herr Peitz,

vielen Dank für Ihre Anfrage.

Wurde nach dem Update der Index neu erstellt, dies kann schon die Ursache für das Verhalten sein. 
Über über ""Administration"" - ""Configuration"" und ""Create new index"" können Sie den ""Index"" der für die Suche benutzt wird neu erstellen lassen.

Viele Grüße,

Christoph Stangl - iteraplan Support"	"17/May/17 2:48 PM;Markus.Bon@inter.de;Hallo Herr Stangl,
hat geklappt, vielen Dank für die schnelle Hilfe :-)

Grüße aus Mannheim
Markus Bon"	17/May/17 2:50 PM;Markus.Bon@inter.de;Wie kann ich das Ticket denn schließen ?	17/May/17 2:53 PM;cst;Ticket geschlossen :)																																																		
